roles/collect/tasks/main.yml

```diff
@@ -28,17 +28,6 @@
     files:
       - '{{ ocp_config["ocp_token_file"] }}'
 
-- name: Check for OC command-line
```

Comment:
Why is this check being removed?

Reply:
I did try to cover the reason for all of my changes in the PR summary above. We now need to expose a route on operator metering to access the API. So we're hitting the API over the internet. No longer a need to login to the cluster.